getting-started/templates/sizing-example/analysis-sizing-example.yaml

```diff
@@ -0,0 +1,217 @@
+## Example configuration for tuning sizing parameters for services related to analysis and automation.
+
+## The JupyterHub IDE
+sl-jupyterhub:
+  jupyterhub:
+    ## Configuration for the JupyterHub coordinator pod.
+    hub:
+      resources:
+        requests:
+          memory: "6Gi"
+          cpu: "1000m"
+        limits:
+          memory: "6Gi"
+
+      ## Database configuration
+      db:
+        pvc:
+          ## @param sl_jupyterhub.jupyterhub.hub.db.pvc.storage Size of the volume where the JupyterHub database is stored.
+          storage: 1Gi
+
+      config:
+        ## jupyter_hub config reference: https://jupyterhub.readthedocs.io/en/stable/api/app.html
+        JupyterHub:
+          ## @param sl_jupyterhub.jupyterhub.hub.config.concurrent_spawn_limit The maximum number of concurrent users that
+          ## can be spawning at a time.
+          concurrent_spawn_limit: 5 # default: 100
+          ## @param sl_jupyterhub.jupyterhub.hub.config.active_server_limit the maximum number of concurrent servers that
+          ## can be active at a time.
+          active_server_limit: 30 # default: 0 - means that no limit is enforced
+
+    ## Configuration for the JupyterHub proxy pod.
+    proxy:
+      resources:
+        requests:
+          memory: "100Mi"
+          cpu: "100m"
+        limits:
+          memory: "100Mi"
+
+    ## Configuration for JupyterHub image puller pods.
+    prePuller:
+      resources:
+        requests:
+          memory: "100Mi"
+          cpu: "100m"
+        limits:
+          memory: "100Mi"
+
+    ## Configuration for individual user pods.
+    singleuser:
+      storage:
+        capacity: 1Gi
+      cpu:
+        limit:
+        guarantee: .5
+      memory:
+        ## NOTE: Size here is in "G" and not "Gi".
+        limit: 2G
+        guarantee: 2G
+
+## Argo workflows is used to schedule notebook executons.
+argoworkflows:
+  argo-workflows:
+    ## Argo workflows controller configuration
+    controller:
+      resources:
+        requests:
+          cpu: 1
+          memory: 4096Mi
+        limits:
+          memory: 4096Mi
+
+      ## @param argoworkflows.argo-workflows.controller.parallelism The maximum number of workflows that can run in parallel.
+      parallelism: &workflowParallelism 300
+
+    ## Argo Workflows Server configuration
+    server:
+      resources:
+        requests:
+          cpu: 250m
+          memory: 1024Mi
+        limits:
+          memory: 1024Mi
+
+      ## @param argoworkflows.argo-workflows.server.replicas The number of server replicas to run.
```

Comment:
Should we state here, as we have in other comments, that this doesn't support HPA?

Reply:
done